### ISSUE TITLE
ML: VGG16 Model #1 (24 Layers): Configuration#9: Training model and store the results in the directory for 30 and 50 epochs

### ISSUE THREAD

**Issue by hmehryar** [hmehryar/Hm.JetscapeMl#273]
Model Architect: 

- 4Conv(Conv+Conv+MaxPool+Dropout)
- 4Dense(Dense+Dropout)

1. Put the training on the Grid by using jetscape-ml-tensorflow-cnn.sh
2. save the model and the evaluation data in the power point [slides](https://docs.google.com/presentation/d/1H7D-TKbmZjwNFNcf8Ecuz3HLl1veCZVL/edit?usp=sharing&ouid=108370064464811889896&rtpof=true&sd=true)
3. update the diagrams data on the power point slides

**Comment by hmehryar** [comment]
30 epochs

Epoch 00030: val_accuracy did not improve from 0.94172

840000/840000 [==============================] - 311s 370us/sample - loss: 0.6347 - accuracy: 0.9403 - val_loss: 0.6239 - val_accuracy: 0.9359

**Comment by hmehryar** [comment]
50 epoch

Epoch 00030: val_accuracy did not improve from 0.94401

840000/840000 [==============================] - 308s 367us/sample - loss: 0.6195 - accuracy: 0.9408 - val_loss: 0.6687 - val_accuracy: 0.9028
Epoch 00030: early stopping

**Comment by hmehryar** [comment]
port into excel for overleaf
